monitoring/bin/create_elasticsearch_datasource.sh

```diff
@@ -181,7 +181,7 @@ else
 fi
 
 # Adds the Elasticsearch data source to Grafana
-log_info "Provisioning Elasticsearch datasource for Grafana"
+log_info "Provisioning Elasticsearch data source for Grafana"
```

Comment:
Drop "Elasticsearch".  Isn't really needed and eliminates a maintenance task when we move to OpenSearch.

Reply:
Since this is the create_elasticsearch_datasource script, I ended up tweaking the verbiage to reference the "Elasticesearch data source in Grafana" instead.
Since the script would require multiple changes, we will create a separate ticket for Opensearch efforts.